packages/d3fc-chart/README.md

```diff
@@ -157,6 +157,29 @@ chart.decorate(selection => {
 });
 ```
 
+#### Using custom axis components
+
+By default the chart uses d3fc-axis for the axis comonent. This example shows how to specify which axis components to use for the X and Y axis.
+
+```javascript
+var chart = fc.chartCartesian({
```

Comment:
const

Reply:
I used `var` because that's what the rest of the ReadMe uses. Would you like me to change them all to `const`?

Reply:
Oh, hadn't realised that. Just stick with `var` it's not a bit deal.